cypress/e2e/patient_spec/patient_crud.cy.ts

```diff
@@ -43,34 +43,22 @@ describe("Patient Creation", () => {
       .then(() => {
         cy.get("[role='option']").contains("Male").click();
       });
-    cy.get("[data-testid=state] button")
-      .click()
-      .then(() => {
-        cy.get("[role='option']").contains("Kerala").click();
-      });
-    cy.get("[data-testid=district] button")
-      .click()
-      .then(() => {
-        cy.get("[role='option']").contains("Ernakulam").click();
-      });
+    cy.get("[data-testid=current-address] textarea").type(
+      "Test Patient Address"
+    );
+    cy.get("[data-testid=permanent-address] input").check();
+    cy.get("#pincode").type("682001");
+    cy.wait(2000);
```

Comment:
@nihal467 instead of hardcoded waits, using intercept would be better. The hardcoded waits may sometimes may not be enough time for the data to be fetched from the API.